nuagekubemon/configure-master.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+RUNAS=root
+if [ "$1" = "k8s" ]; then
+    BIN=/usr/bin/nuagekubemon
+    PIDFILE=/var/run/nuagekubemon.pid
+    rm -irf /usr/bin/nuage-openshift-monitor
+    MONITOR=nuagekubemon
+    TMP_CONF='/tmp/nuagekubemon.yaml'
+    MONITOR_CONF='/usr/share/nuagekubemon/nuagekubemon.yaml'
+fi
+
+if [ "$1" = "ose" ]; then
+    BIN=/usr/bin/nuage-openshift-monitor
+    PIDFILE=/var/run/nuage-openshift-monitor.pid
+    rm -irf /usr/bin/nuagekubemon
+    MONITOR=nuage-openshift-monitor
+    TMP_CONF='/tmp/nuage-openshift-monitor.yaml'
+    MONITOR_CONF='/usr/share/nuage-openshift-monitor/nuage-openshift-monitor.yaml'
+fi
+
+if [ "$2" = "is_atomic" ]; then
+    DIR='/var/usr/share'
+    MONITOR_CONF=$DIR/$MONITOR/$MONITOR.yaml
+fi
+
+# Configuring Nuage monitor yaml file on master nodes using
+# daemon sets
+cat > $TMP_CONF << EOF
```

Comment:
Any reason why we are using TMP_CONF? we can as well write directly to MONITOR_CONF.. am i missing something?